config/Makefile.depend

```diff
@@ -332,9 +332,10 @@ MLPLIBEXTRA             = lib/libHist.lib lib/libMatrix.lib lib/libTree.lib \
                           lib/libTreePlayer.lib lib/libMathCore.lib
 SPECTRUMLIBEXTRA        = lib/libHist.lib lib/libMatrix.lib
 UNFOLDLIBEXTRA          = lib/libHist.lib lib/libMatrix.lib lib/libXMLParser.lib
-TMVALIBEXTRA            = lib/libRIO.lib lib/libHist.lib lib/libMatrix.lib \
-                          lib/libTree.lib lib/libGraf.lib lib/libGpad.lib \
-                          lib/libTreePlayer.lib  lib/libMLP.lib \
+TMVALIBEXTRA            = lib/libCore.lib lib/libNet.lib lib/libMultiProc.lib \
+                          lib/libRIO.lib lib/libHist.lib lib/libMatrix.lib \
+                          lib/libTree.lib lib/libGraf.lib lib/libGpad.lib lib/libNet.lib \
```

Comment:
libNet show up twice

Reply:
`libNet` became a dependency of multiproc, so probably the order matters (see commit dbadc9050a965f062bf1848653d79bdb80177fb9). I will remove the second ocurrence.